src/lib/eliom_uri.shared.ml

```diff
@@ -18,8 +18,28 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 *)
 
-(*****************************************************************************)
 (* Building href *)
+
+let app_path = ref None
```

Comment:
If I'm right, this reference is only used on client side. Either use a `let%client` or, if you can, add a comment.

If it is server side, should probably depend on the `<site>` in config file, not be a global reference.
(We can have several Eliom applications on the same server.) 
→ put this in `sitedata`.

(Don't we already have this somewhere in sitedata?)